posthog/settings.py

```diff
@@ -53,6 +53,8 @@ def get_env(key):
     USE_X_FORWARDED_HOST = True
     SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
+ALLOWED_IP_BLOCKS = os.environ.get('ALLOWED_IP_BLOCKS', False)
```

Comment:
I think it would be very useful to allow subnets. The current regexp implementation kind of allows it... but it has the other problem discussed above.

What about instead having `ALLOWED_IP_BLOCKS` be a List of ip addresses or ip address ranges which get passed through the [python standard `ipaddress` library](https://docs.python.org/3/library/ipaddress.html)? 

Then if any of the IP blocks match the request's ip, access granted.

```python
addr = ip_address('10.0.1.0')
netw = ip_network('123.1.0.0/8', strict=False)
print(addr in netw) # allowed
```

Reply:
Great idea on the `subnet` thing.

This works great for hardcoded values in `settings.py`, but how do we make it work with environment variables?

A simple comma-separated string could be allowed, but again that comes with some added code for parsing such a string.

Reply:
That doesn't sound too hard :)

Reply:
Thanks both, absolutely right. My original implementation wasn't good enough. Updated as per Marius' suggestion, should be a lot more obvious and more useful now!